api/HTMLIFrameElement.json

```diff
@@ -0,0 +1,2454 @@
+          },
+          "firefox_android": {
+            "version_added": "17",
+            "notes": "Previously, the type of <code>sandbox</code> was a <a href='https://developer.mozilla.org/docs/Web/API/DOMString' title='DOMString is a UTF-16 String. As JavaScript already uses such strings, DOMString is mapped directly to a String.'><code>DOMString</code></a> instead of a <a href='https://developer.mozilla.org/docs/Web/API/DOMSettableTokenList' title='The documentation about this has not yet been written; please consider contributing!' class='new' rel='nofollow'><code>DOMSettableTokenList</code></a>. This has been fixed with <a href='https://developer.mozilla.org/Firefox/Releases/29/Site_Compatibility'>Gecko&nbsp;29.0</a> (Firefox 29 / Thunderbird 29 / SeaMonkey 2.26)) (<a href='https://bugzil.la/845067' title='Send Email screen update'>bug&nbsp;845067</a>). Other browsers may still implement the property as <code>DOMString</code> since it was a late change in the specification."
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
+    },
+    "scrolling": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/scrolling",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": true
+        }
+      }
+    },
+    "src": {
```

Comment:
I don't think `src` is any longer part of the `HTMLIFrameElement` API https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/src

I'm not finding any data on it.